pinax/blog/models.py

```diff
@@ -189,14 +193,22 @@ def save(self, **kwargs):
         super(Post, self).save(**kwargs)
 
     @property
+    def scoping_url_kwargs(self):
```

Comment:
There is an issue with this method only being available on `Post`. The scoper is now the "blog" object (due the absence of an actual `Blog` that would do the scoping). I suspect you will run into an issue in places where you are working with a list of posts.